tier3/checklist.md

```diff
@@ -0,0 +1,441 @@
+# DSAC OSPO Outbound Review Checklist
+## Tier 3: Public Repository
+### Instructions
+This is a review process to approve CMS-developed software to be released open source.
+If you would like your repository to be released, please complete the following steps.
+
+[Instructions](#instructions)
+
+[State the Benefit(s) of Open Sourcing the Project](#state-the-benefits-of-open-sourcing-the-project)
+
+[State the Risk(s) of Open Sourcing the Project, if any](#state-the-risks-of-open-sourcing-the-project-if-any)
+
+[Questions](#questions)
+
+[Code Review](#code-review)
+
+[Code Analysis](#code-analysis)
+
+[Toolkit](#toolkit)
+
+[Review Licensing](#review-licensing)
+
+[Review Commit History](#review-commit-history)
+
+[Review Repository Hygiene](#review-repository-hygiene)
+
+[Running repolinter on your repository locally](#running-repolinter-on-your-repository-locally)
+
+[Running repolinter on your repository via GitHub Actions](#running-repolinter-on-your-repository-via-github-actions)
+
+[Review Content](#review-content)
+
+[Review Repository Details](#review-repository-details)
+
+[Additional Notes & Questions](#additional-notes--questoins)
+
+[Sign off on risk acceptance of open-sourcing the software product](#sign-off-on-risk-acceptance-of-open-sourcing-the-software-product)
+
+[Flipping the Switch: Making the Repository Public](#flipping-the-switch-making-the-repository-public)
+
+[Repository Actions](#repository-actions)
+
+[Communications & Rollout 📣](#communications--rollout-📣)
+
+[Tracking 📈](#tracking-📈)
+
+[Flipping the Switch: Making the Repository Public](#flipping-the-switch-making-the-repository-public)
+
+
+
+
+
+### State the Benefit(s) of Open Sourcing the Project
+
+- [ ] Cost Savings
```

Comment:
Please bold each of the items in this list: "Cost Savings", "Ease of Repurposing", etc.